File: src/test/java/bad/robot/radiate/NullProgressTest.java
package bad.robot.radiate;

import org.junit.Test;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.is;

@Deprecated
public class NullProgressTest {

    @Test
    public void basicEquality() {
        assertThat(new NullProgress().equals(new NullProgress()), is(true));
    }

    @Test
    public void showTheNumberOfThingsProgressIsOver() {
        assertThat(new NullProgress().numberOfBuilds(), is(0));
    }

}

File: src/test/scala/bad/robot/radiate/NullProgressTest.scala
package bad.robot.radiate

import org.specs2.mutable.Specification

class NullProgressTestS extends Specification {
  
  "Basic equality" >> {
    new NullProgress() must_== new NullProgress()
  }

  "Show the number of things progress is over" >> {
    new NullProgress().numberOfBuilds must_== 0
  }
}